frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -56,7 +56,7 @@ const FormContentWrapper = styled.div`
   flex-direction: column;
```

Comment:
Note that the side bar was trying to use box display to handle the 100% vh of the form content. This was actually setting the form to 100% of the sidebar, even though other content above the form was present, meaning the form + other content was > 100% of the parent.

Switched the parent to a flex, this ensures that the children will not exceed the height of the parent. No need for really long margins anymore to get around some of the form being outside the parent. Need slightly awkward negative absolute position now to get around the padding of the parent, but still seems much better overall.

The scroll bar now displays correctly as well (was getting cut off the bottom of the screen before).